tests/rkt_volume_test.go

```diff
@@ -77,22 +77,24 @@ func TestVolumes(t *testing.T) {
 	defer os.Remove("rkt-inspect-vol-ro-read-file.aci")
 	patchTestACI("rkt-inspect-vol-ro-write-file.aci", "--exec=/inspect --write-file --read-file", "--mounts=dir1,path=/dir1,readOnly=true")
 	defer os.Remove("rkt-inspect-vol-ro-write-file.aci")
+	ctx := newRktRunCtx()
+	defer ctx.cleanup()
 
 	tmpdir, err := ioutil.TempDir("", "rkt-tests.")
 	if err != nil {
 		t.Fatalf("Cannot create temporary directory: %v", err)
 	}
-	defer os.Remove(tmpdir)
+	defer os.RemoveAll(tmpdir)
 
 	tmpfile := filepath.Join(tmpdir, "file")
 	err = ioutil.WriteFile(tmpfile, []byte("host"), 0600)
 	if err != nil {
 		t.Fatalf("Cannot create temporary file: %v", err)
 	}
-	defer os.Remove(tmpfile)
 
 	for i, tt := range volTests {
 		cmd := strings.Replace(tt.rktCmd, "$TMPDIR", tmpdir, -1)
+		cmd = strings.Replace(cmd, "^RKT_BIN^", ctx.cmd(), -1)
```

Comment:
Since you seem to like the `^FOO^` pattern, maybe the previous line should be `^TMPDIR^` for consistency? In this way, it will be clearer that `$TMPDIR` is not an environment variable.